ci-operator/step-registry/ipi/conf/vsphere/folder/ipi-conf-vsphere-folder-commands.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+echo "$(date -u --rfc-3339=seconds) - sourcing context from vsphere_context.sh..."
+# shellcheck source=/dev/null
+declare vsphere_datacenter
+# shellcheck disable=SC1091
+source "${SHARED_DIR}/vsphere_context.sh"
+echo "$(date -u --rfc-3339=seconds) - Configuring govc exports..."
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/govc.sh"
+
+FOLDER="/$vsphere_datacenter/vm/$vsphere_datacenter"
+SUB_FOLDER="$FOLDER/${NAMESPACE}-${UNIQUE_HASH}"
+
+if govc folder.info "$SUB_FOLDER"; then
+  echo "$SUB_FOLDER already exist, no need to create"
+else
+  if govc folder.info "$FOLDER"; then
+    echo "$FOLDER already exist, no need to create"
+  else
+    govc folder.create "$FOLDER"
+  fi
+  govc folder.create "$SUB_FOLDER"
```

Comment:
I remembered that installer doesn't delete customer folder when destroying cluster,  so we might need to consider deleting sub folder in deprovision stage, else left over empty folders which could not be reused any more.

Reply:
Good catch!

Reply:
I can delete "$SUB_FOLDER" when de-provision, but better to keep $FOLDER in case multy jobs run.

Reply:
Why not directly use the existing folder - `/$vsphere_datacenter/vm` as `FOLDER` ?

Reply:
In case, there's a check point
```
Note: per OCPBUGS-7063, let folder path contain datacenter name
```

Reply:
good to know.